ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
Not sure about that. This test is specifically for isClosed method. Close method is tested by closeConnectionInAfterCompletion.
I did that on purpose to make test units as small as possible.